### PR TITLE
re-write forms with react-broadcast

### PR DESCRIPTION
**Pull request by ramitos**

### CODE REVIEW

**Comment by ramitos** [comment]
don't merge (yet)

File: frontend/src/components/new-project/index.js [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
 const constants = require('@ui/shared/constants');
 const fns = require('@ui/shared/functions');
-
-const Input = require('@ui/components/input');
 const Button = require('@ui/components/button');
 const BaseElements = require('@ui/components/base-elements');
+const Form = require('@ui/components/form');
 
 const {
   H2,
```
Comment (alex-windett) [inline frontend/src/components/new-project/index.js]: Trailing comma will fail in my linting update

File: frontend/src/components/new-project/new-billing.js [hunk 2]
```diff
@@ -1,22 +1,24 @@
...
-
-const Input = require('@ui/components/input');
-const Button = require('@ui/components/button');
 const BaseElements = require('@ui/components/base-elements');
+const normalize = require('@root/utils/form/normalize');
+const Form = require('@ui/components/form');
+const Button = require('@ui/components/button');
 
 const {
   H2,
```
Comment (alex-windett) [inline frontend/src/components/new-project/new-billing.js]: And again

File: frontend/src/containers/metrics/monitors.js [hunk 3]
```diff
@@ -8,9 +8,9 @@ const ManageMonitor = require('@components/manage-monitor');
 const Monitors = require('@components/monitors');
 const selectors = require('@state/selectors');
 
-const {
+/*const {
   reduxForm
-} = ReduxForm;
+} = ReduxForm;*/
```
Comment (alex-windett) [inline frontend/src/containers/metrics/monitors.js]: Remove comment ?? Or is it going to be useful later and also on line 28

File: frontend/src/components/people-list/table/index.js [hunk 4]
```diff
@@ -18,6 +18,11 @@ const {
...
 } = fns;
 
+const {
+  Checkbox,
+  FormGroup
+} = Form;
+
 const PeopleTable = (props) => {
 
   const {
```
Comment (alex-windett) [inline frontend/src/components/people-list/table/index.js]: Empty object de-structuring

File: ui/src/components/form/label.js [hunk 5]
```diff
@@ -1,33 +1,43 @@
...
-const fns = require('../../shared/functions');
+const React = require('react');
+const ReactBroadcast = require('react-broadcast');
 const Styled = require('styled-components');
+const fns = require('../../shared/functions');
 
 const {
-  colors
-} = constants;
+  default: styled,
```
Comment (alex-windett) [inline ui/src/components/form/label.js]: Remove trailling comma

File: ui/src/components/table-simple-table/table.js [hunk 6]
```diff
@@ -24,16 +24,18 @@ const StyledTableWrapper = styled.section`
...
+  <StyledTableWrapper style={style} className={className}>
     {children}
   </StyledTableWrapper>
 );
 
 Table.propTypes = {
   children: React.PropTypes.node,
+  className: React.PropTypes.string,
   style: React.PropTypes.object,
   title: React.PropTypes.string,
```
Comment (alex-windett) [inline ui/src/components/table-simple-table/table.js]: Trailing comma

File: ui/src/index.js [hunk 7]
```diff
@@ -4,29 +4,28 @@ module.exports = {
...
+  Select: require('./components/form/select'),
   SelectCustom: require('./components/select-custom'),
   Tab: require('./components/tabs/tab'),
   Tabs: require('./components/tabs'),
   Toggle: require('./components/toggle'),
   Topology: require('./components/topology'),
   Tooltip: require('./components/tooltip'),
-  Textarea: require('./components/textarea'),
   Widget: require('./components/widget'),
   BaseElements: require('./components/base-elements'),
```
Comment (alex-windett) [inline ui/src/index.js]: Trailing comma

File: frontend/src/containers/metrics/monitors.js (hunk 3 above)
Comment (ramitos) [inline frontend/src/containers/metrics/monitors.js]: yes, that is going to be used when create-monitor is an actual form

**Pull request merged**